### DISCUSSION TITLE
error when loading ttf font

### DISCUSSION THREAD

**Discussion by lbr991** [pymupdf/PyMuPDF#2724] [Looking for help] [answered]
I'm using PyMuPDF 1.23.4 and I'm using insert_text with a ttf font:
`my_font = fitz.Font(fontfile="tahoma.ttf")`
I have copied tahoma.ttf in the current dir.

`rc = page.insert_text((x0, y0), word, fontname=my_font, color=(1, 0, 0), fontsize=fontsize)`

I'm getting this error:
```
    rc = img.insert_text(
  File "C:\Users\user\anaconda3\lib\site-packages\fitz\utils.py", line 3460, in insert_text
    if fname.startswith("/"):
AttributeError: 'Font' object has no attribute 'startswith'
```

**Comment by JorjMcKie** [comment]
fontname is not the filename!
It is the internal reference of the font. In essence you are free to select an arbitrary one like "lbr991" - independent from the name of the font file.
Exceptions are the Base14 reference names like "helv", "tiro" and so on.

**Reply by lbr991** [reply]
How can I load and use a specific .ttf font?

**Reply by lbr991** [reply]
Sorry, got it, thanks.

**Reply by JorjMcKie** [reply, marked as answer]
Specify fontfile **_and_** fontname.
